packages/cli/src/__tests__/commands/Version.test.ts

```diff
@@ -112,17 +112,15 @@ function cleanSnapshot(str: string, versionOverride?: string): string {
   //                                                                                    ^^^^^^^^^^^^^^^^^^^
   str = str.replace(/\(at (.*engines)(\/|\\)/g, '(at sanitized_path/')
 
-  // TODO: replace '[a-z0-9]{40}' with 'ENGINE_VERSION'.
-  // Currently, the engine version of @prisma/prisma-fmt-wasm isn't necessarily the same as the enginesVersion
```

Comment:
cc @jkomyno do you know why they aren't necessarily the same versions?

Reply:
Technically one could use a different one if there are not relevant changes that affect prisma-fmt/schema-wasm maybe, but I don't think our setup is designed to ever actually do that. They _should_ be in sync. (cc @millsp)